utils/docker.go

```diff
@@ -191,6 +196,16 @@ func (u *Utility) RunContainer(image string, ports []string, volumes []string, c
 
 }
 
+func (u *Utility) validateRunContainerVolume(volume string) error {
+	splitVolume := strings.Split(volume, ":")
+
+	if len(splitVolume) != 2 && len(splitVolume) != 3 {
+		return fmt.Errorf("utils: Invalid split volume %d", len(splitVolume))
```

Comment:
I like this error message, but can we add the word "length" in it?

Something like:
`utils: Invalid split volume of length %d`

Reply:
@everettraven 
Ok, I will update that one.

Reply:
@everettraven 
When the corresponding change has been pushed, should I hit the "Resolve conversation" button? Or is there any other thing that I should do? (Sorry, I am new to this platform) Thanks.

Reply:
@neverbeenthisweeb You can hit the resolve conversation button whenever you make the change. I will go ahead and resolve it though

Reply:
@everettraven 
Ok, got it.